src/common/cockpitloopback.c

```diff
@@ -84,12 +83,7 @@ cockpit_loopback_next_finish (GSocketAddressEnumerator *enumerator,
                               GAsyncResult *result,
                               GError **error)
 {
-  GSocketAddress *address;
```

Comment:
No `g_task_is_valid()` here?

Reply:
The original code doesn't have it, but indeed it seems cleaner to assert it. Added.